### PR TITLE
ZipFile: Added support for multiple file zips

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related to: https://github.com/demisto/etc/issues/30036

## Description
Added the option to pass a list of entryIDs and zip them together.

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by Itay4** [comment]
please add a test to the test playbook

File: Packs/CommonScripts/Scripts/ZipFile/ZipFile.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 args:
 - default: false
-  description: entry id of the file we want to zip
-  isArray: false
+  description: CSV list of entry ids of the files we want to zip.
```
Comment (amshamah419) [inline Packs/CommonScripts/Scripts/ZipFile/ZipFile.yml:3]:
```suggestion
  description: CSV list of entry ids for the files to zip.
```

Comment (amshamah419) [reply Packs/CommonScripts/Scripts/ZipFile/ZipFile.yml:3]: Or disregard, I didn't see @roysagi 's review

**Comment by Itay4** [comment]
@jochman please update the script readme as well

**Review by Itay4 — APPROVED** [review]

**Pull request merged**